x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/knowledge_base/knowledge_base.spec.ts

```diff
@@ -200,6 +209,121 @@ export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderCon
       });
     });
 
+    describe('when managing multiple entries across spaces', () => {
+      const SPACE_A_ID = 'space_a';
+      const SPACE_A_NAME = 'Space A';
+      const SPACE_B_ID = 'space_b';
+      const SPACE_B_NAME = 'Space B';
```

Comment:
For simplicity let's just have id and not also name. 
```suggestion
      const SPACE_A_ID = 'space_a';
      const SPACE_B_ID = 'space_b';
```

Reply:
Sure. I updated it